WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -118,6 +122,11 @@ sealed class PostListItemViewHolder(
         showFeaturedImage(data.imageUrl)
         updateProgressBarState(data.progressBarState)
         uiHelpers.updateVisibility(disabledOverlay, data.showOverlay)
+        if (data.disableRippleEffect) {
+            container.background = null
+        } else {
+            container.background = getDrawableFromAttribute(container.context, android.R.attr.selectableItemBackground)
```

Comment:
NitPick: What do you think about caching the drawable in a field so we don't load it whenever the item gets recycled?

Reply:
Good point, I forgot all view holder good practices :) - fixed in 1886399b8b49cab120dd61275869922c32f3a6ae